### DISCUSSION TITLE
I want to join Github Education

### DISCUSSION THREAD

**Discussion by Pma10** [community/community#75220] [GitHub Education]
### Hi everyone,

I wrote it myself because my school wasn't on the list, and I also used Billing as my name, and I took my student ID and sent it over several times, but all of these applications were rejected, and now it's automatically rejected without being reviewed by anyone. It's unfair for me. I've tried everything I can and I keep getting rejected. What should I do.

**Comment by NotIMaestro** [comment]
In the same boat, i study in a non-english school and i feel like my applications get automatically denied because all the documents are in a different language

**Reply by Pma10** [reply]
I put a post-it on my student id and submitted my name in English, and it passed!
